assets/js/sections/qso.js

```diff
@@ -1019,18 +1024,31 @@ $("#callsign").on("keypress", function(e) {
 // On Key up check and suggest callsigns
 $("#callsign").keyup(function() {
 	if ($(this).val().length >= 3) {
-	  $('.callsign-suggest').show();
-	  $callsign = $(this).val().replace('Ø', '0');
-	  $.ajax({
-		url: 'lookup/scp',
-		method: 'POST',
-		data: {
-		  callsign: $callsign.toUpperCase()
-		},
-		success: function(result) {
-		  $('.callsign-suggestions').text(result);
-		}
-	  });
+	  $callsign = $(this).val().replace('Ø', '0').toUpperCase();
```

Comment:
a few things:

1. isn't using `$` on the var a php thing? This is a js variable. 
2.  can it match the same variable name as the contesting.js file? 
3.  is the `.replace('Ø', '0')` only required for qso and not contesting?

Reply:
The code just reuses existing variables. `$` is eligible in JS-vars (heavily used by jQuery, e.g. ;) ).

I'll have a look on the replace as it poses 'issues' :D.

Reply:
its a nit for sure, but i feel like we should try our best to be consistent with modern code patterns :)

Reply:
@magicbug should the `.replace('Ø', '0')` function call also be moved to the contesting.js logic?  Or is it only necessary for regular QSO's?

Reply:
Please see @magicbug s comment about '2E0' and my response below.

Reply:
i saw both, neither seemed directly related to my question regarding whether the logic belonged in both places

Reply:
I considered it 'wip' with Peter's comment already... :)